x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/oidc/OpenIdConnectRealmSettings.java

```diff
@@ -212,6 +213,12 @@ private OpenIdConnectRealmSettings() {}
         "http.max_endpoint_connections",
         key -> Setting.intSetting(key, 200, Setting.Property.NodeScope)
     );
+
+    public static final Setting.AffixSetting<TimeValue> HTTP_CONNECTION_POOL_TTL = Setting.affixKeySetting(
+        RealmSettings.realmSettingPrefix(TYPE),
+        "http.connection_pool_ttl",
```

Comment:
I wonder whether this should be
```suggestion
        "http.connection_pool.ttl",
```

Reply:
Or even
```suggestion
        "http.connection_pool.idle_timeout",
```

Reply:
This name is taken after the similar [watcher setting](https://www.elastic.co/guide/en/elasticsearch/reference/current/notification-settings.html#general-notification-settings). It's probably better to keep consistent?

Reply:
Fair enough